dts/bindings/lora/semtech,sx1276.yaml

```diff
@@ -24,3 +24,8 @@ properties:
         Up to six pins that produce service interrupts from the modem.
 
         These signals are normally active-high.
+
+    lora-rtcs:
+      type: phandle-array
```

Comment:
This should be `phandles` not `phandle-array`.  Also, what does it mean if someone sets this to multiple RTCs?

Reply:
phandles is not generated by the python scripts. @ulfalizer suggested using the array, where one instance is just a special case of an array.

Reply:
Let's figure out what it should be and then we can figure out what works right now.

Do you realistically see need more than one RTC?  Should this really just be a phandle? (singleton)

Reply:
@Mani-Sadhasivam what do you think? IMO we only need one.

Reply:
@alexanderwachter Yes, we need only one RTC for LoRa. Initially, I got into the assumption that you've used `phandle-array` because of zephyr limitation. But if we can use just a phandle for this, I would favor.

Reply:
I also looked into putting the RTC into dts and phandle indeed seems to be not available (see `scripts/dts/gen_defines.py:268`). Using phandle-array is a workaround for this.

Reply:
@ulfalizer can you look at adding phandle(s) support to get_defines.py

Reply:
What macros do you need to be generated for the pointed-to nodes in the case of `phandle` and `phandles`? They're just generic node references in themselves, so will have to sort that out first.

Reply:
See PR #22770 for how I think we should handle phandle(s) generation.